includes/admin/payments/refunds.php

```diff
@@ -0,0 +1,200 @@
+			'cartIndex'    => esc_html( $item->cart_index ),
+			'type'         => esc_html( $item->type ),
+			'status'       => esc_html( $item->status ),
+			'quantity'     => esc_html( $item->quantity ),
+			'amount'       => esc_html( $item->amount ),
+			'subtotal'     => esc_html( $item->subtotal ),
+			'discount'     => esc_html( $item->discount ),
+			'tax'          => esc_html( $item->tax ),
+			'total'        => esc_html( $item->total ),
+			'dateCreated'  => esc_html( $item->date_created ),
+			'dateModified' => esc_html( $item->date_modified ),
+			'uuid'         => esc_html( $item->uuid ),
+		);
+	}
+
+	wp_localize_script(
+		'edd-admin-orders',
+		'eddAdminOrderOverview',
+		array(
+			'items'        => $_items,
+			'adjustments'  => array(),
+			'refunds'      => array(),
+			'isAdding'     => false,
+			'isRefund'     => true,
+			'hasQuantity'  => true === edd_item_quantities_enabled(),
+			'hasTax'       => true === edd_use_taxes()
+				? array(
+					'rate'    => 0,
+					'country' => '',
+					'region'  => '',
+				)
+				: 0,
+		)
+	);
+
+	$templates = array(
+		'totals',
+		'item',
+		'adjustment',
+		'adjustment-discount',
+	);
+
+	foreach ( $templates as $tmpl ) {
+		echo '<script type="text/html" id="tmpl-edd-admin-order-' . esc_attr( $tmpl ) . '">';
+		require_once EDD_PLUGIN_DIR . 'includes/admin/views/tmpl-order-' . $tmpl . '.php';
+		echo '</script>';
+	}
+?>
+
+<div id="edd-order-overview" class="postbox edd-edit-purchase-element edd-order-overview">
+	<table id="edd-order-overview-summary" class="widefat wp-list-table edd-order-overview-summary edd-order-overview-summary--refund">
+		<thead>
+			<tr>
+				<th class="column-name column-primary"><?php echo esc_html( edd_get_label_singular() ); ?></th>
+				<th class="column-amount"><?php esc_html_e( 'Unit Price', 'easy-digital-downloads' ); ?></th>
+				<?php if ( true === edd_item_quantities_enabled() ) : ?>
+				<th class="column-quantity"><?php esc_html_e( 'Quantity', 'easy-digital-downloads' ); ?></th>
+				<?php endif; ?>
+				<th class="column-subtotal column-right"><?php esc_html_e( 'Amount', 'easy-digital-downloads' ); ?></th>
+			</tr>
+		</thead>
+	</table>
+</div>
+
+<?php
+}
+
+/**
+ * Outputs Refund Notes box.
+ *
+ * @since 3.0
+ *
+ * @param \EDD\Orders\Order $refund Current Refund.
+ */
+function edd_refund_details_notes( $refund ) {
+?>
+
+<div id="edd-order-items" class="postbox edd-edit-purchase-element">
+	<h3 class="hndle">
+		<span><?php esc_html_e( 'Refund Notes', 'easy-digital-downloads' ); ?></span>
+	</h3>
+
+	<div class="inside">
+		<?php edd_order_details_notes( $refund ); ?>
+	</div>
+</div>
+
+<?php
+}
+
+/**
+ * Outputs Refund Attribuets box.
+ *
+ * @since 3.0
+ *
+ * @param \EDD\Orders\Order $refund Current Refund.
+ */
+function edd_refund_details_attributes( $refund ) {
+	$refund_date = edd_get_edd_timezone_equivalent_date_from_utc( EDD()->utils->date( $refund->date_created, 'utc', true ) );
+?>
+
+<div class="postbox">
+	<h3 class="hndle">
```

Comment:
See the heading/class/span notes above.

Also not sure about the wording, "Refund Attributes", but not sure I have a better suggestion. Maybe "Details"?